src/components/LoansByCategory/FeaturedHeroLoan.vue

```diff
@@ -173,6 +194,24 @@ export default {
 			}
 			return ' ';
 		},
+		expiringSoonMessage() {
```

Comment:
in the next task for this to use the LoanCardController, you'll get to remove all of this as it's provided for you.